packages/frontend/src/features/scheduler/components/SchedulerForm.tsx

```diff
@@ -343,6 +358,11 @@ const SchedulerForm: FC<Props> = ({
                         <Tabs.Tab value="filters">Filters</Tabs.Tab>
                     ) : null}
                     <Tabs.Tab value="customization">Customization</Tabs.Tab>
+                    {form.values.format === SchedulerFormat.IMAGE && (
```

Comment:
Maybe we should disable this tab in this case instead of hiding it? It feels a little odd when it pops in.

Reply:
Done 👍